cmd/scan.go

```diff
@@ -25,13 +25,15 @@ var debug bool
 var absolutePath bool
 var format string
 var template string
+var schemaDir string
 var outputLocation string
 var exitCode int
 
 func init() {
 	scanCmd.Flags().BoolVar(&debug, "debug", false, "turn on debug logs")
 	scanCmd.Flags().BoolVar(&absolutePath, "absolute-path", false, "use the absolute path for the file name")
 	scanCmd.Flags().StringVarP(&format, "format", "f", "json", "Set output format (json, template)")
+	scanCmd.Flags().StringVarP(&schemaDir, "schema-dir", "schema-dir", "", "Sets the directory for the json schemas")
```

Comment:
Sorry, can you use `StringVar` when there's no shorthand

```suggestion
	scanCmd.Flags().StringVar(&schemaDir, "schema-dir", "", "Sets the directory for the json schemas")
```

---

Also your commit isn't tying to your GH account, so you might want to check you're using the right email.
It's not anything that'd prevent merging but it can be important if you want to show up on the contributers section etc :sweat_smile:

Reply:
Thx for pointing out that I used an other email. I'll change that.

Reply:
I've updated the email by rewriting my commits. I hope that is ok in the context of a pull request.